deploy/before_restart

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+oldrev, newrev = ARGV
+
+def run(cmd)
+  exit($?.exitstatus) unless system "umask 002 && #{cmd}"
+end
+
+RAILS_ENV   = ENV['RAILS_ENV'] || 'production'
```

Comment:
I have a concern about doing the fallback to production like this - what if the `RAILS_ENV` is set to an empty string because of an error in a ansible or salt script, e.g:

``` irb
$ RAILS_ENV= irb
>> ENV['RAILS_ENV'] || 'production'
=> ""
```

Not sure what the answer is :smile: